Formula/ansible-lint.rb

```diff
@@ -3,8 +3,8 @@ class AnsibleLint < Formula
 
   desc "Checks ansible playbooks for practices and behaviour"
   homepage "https://ansible-lint.readthedocs.io/"
-  url "https://files.pythonhosted.org/packages/07/3e/16ee36193be535dbd95e16bd0a668de58367fb0ff3a7759e47e7b954cb78/ansible-lint-6.14.6.tar.gz"
-  sha256 "2a364458ae46c8edc3d6b5a78ac573dc1a49eec64c856c85a2cb205ea3d38a2e"
+  url "https://github.com/ansible/ansible-lint/archive/v6.15.0.tar.gz"
```

Comment:
Please keep using the pypi URL. `brew bump-formula-pr` will also be able to update the resources when using that URL.